crates/fe/src/main.rs

```diff
@@ -77,6 +77,12 @@ pub fn main() {
                 .help("dump mir dot file")
                 .takes_value(false),
         )
+        .arg(
+            Arg::with_name("codegen")
+                .long("codegen")
+                .help("todo")
```

Comment:
That's a user facing `todo` so it should be replaced with something meaningful.

Reply:
Now `--codegen` is no longer necessary, so I just removed the argument.